js/main.js

```diff
@@ -1,2 +1,16 @@
-import './data.js';
-import './render-random-offers.js';
+
+import { getOfferCard } from './card.js';
+
+const mapCanvas = document.querySelector('.map__canvas');
+const randomOffersFragment = document.createDocumentFragment();
+
+const renderOffers = (number) => {
+  for (let i = 1; i <= number; i++) {
+    randomOffersFragment.appendChild(getOfferCard());
+  }
+}
+
+renderOffers(1);
+mapCanvas.appendChild(randomOffersFragment);
```

Comment:
Предлагаю отказаться от замыкания и перенести фрагмент в саму функцию. Также обрати внимание на нейминг: в большинстве случаев глагол **render** используется в контексте непосредственной отрисовки на странице, тут же мы лишь генерим объявления
```suggestion
const generateOffers = (amount) => {
  const randomOffersFragment = document.createDocumentFragment();
  for (let i = 1; i <= amount; i++) {
    randomOffersFragment.appendChild(getOfferCard());
  }
  return randomOffersFragment;
}

mapCanvas.appendChild(generateOffers(1));
```